.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run: node -v
+      - run: npm -v
```

Comment:
`yarn versions` gives you the versions of Node, Yarn and built-in module versions in one command.